kie-server-parent/kie-server-tests/kie-server-integ-tests-drools/src/test/java/org/kie/server/integrationtests/drools/pmml/ApplyScorecardMultiModuleIntegrationTest.java

```diff
@@ -39,7 +39,7 @@
     @BeforeClass
     public static void buildAndDeployArtifacts() {
         KieServerDeployer.buildAndDeployCommonMavenParent();
-        KieServerDeployer.buildAndDeployMavenProjectFromResource("/kjars-sources/scorecard");
```

Comment:
I know that it is nitpicking and that I originally coded it this way, but we might want to make the path for the resource into a `private static final String`

Reply:
Just realized that I put the above comment on the line before the one I intended request for change.

Reply:
Adjusted